### DISCUSSION TITLE
Download Failed for Chrome

### DISCUSSION THREAD

**Discussion by jhagins45** [aclap-dev/video-downloadhelper#1348] [Help and Q&A]
### Your question

I use a website Moviepire.net and it always works and have no issues but for some reason now when I go to download any video on the chrome extension, I get the error message "Download Failed: Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)." 

Is there anything that can be done? I downloaded the coapp and it works for other websites on chrome too. I also uninstalled and reinstalled the extension and restarted the browser and the computer and it still doesn't work. 

Website I used for example: https://moviepire.net/watch/127858?s=1&e=1

### Addon & CoApp version and details

I use a website Moviepire.net and it always works and have no issues but for some reason now when I go to download any video on the chrome extension, I get the error message "Download Failed: Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)." 

Is there anything that can be done? I downloaded the coapp and it works for other websites on chrome too. I also uninstalled and reinstalled the extension and restarted the browser and the computer and it still doesn't work.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

First off, when you create a problem report here, there are instructions on that page.  Those instructions are not there just to take up space.  You're supposed to follow the instructions.  It is important to post the information about your environment as instructed there.  If you wish to abbreviate those instructions, you should do it like this:

Windows 11 Pro 64-bit version 24H2 build 26100.2033, Firefox 131.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20

If you ever open another problem report here, follow the instructions.

I visited the web page whose URL you posted.  VDH recognized nothing at first.  After I clicked the Play button in the player, playback did not launch.  It just showed a Play button again.  However, that was enough to get VDH to recognize something.  But when I tried to download it, I almost immediately got the annoying & overly cryptic error message asking me to report the problem.  I've complained about that error message before.  It's woefully inadequate as an error message.  It should give us a chance to see the information that will be reported when we click the report button.

I also looked in the Firefox Network Monitor to see what was there.  There was an HLS master manifest.  But when I tried to analyze it with ffprobe, I got a 403 Forbidden error.  I suspect this is what is blocking VDH, that the HLS master manifest is somehow protected against reading.  My conclusion is that the web site wants to thwart downloaders.

You say this used to work.  Clearly, it doesn't work now.  That has to be due to changes the web site made.  This sort of thing happens all the time.  VDH used to work on a site but suddenly the site decides to be annoying & block downloads that used to work.  It's like an epidemic.  Web sites are doing things like what this site is doing.  Either that or they are applying DRM protection.  There's no way around it.  This site does not have DRM in place.  That means there is a chance that a screen recorder could record the content.  In fact, I did record a few seconds of this with OBS as a test & it worked fine.  Now, screen recording is definitely not as good as downloading.  You have to let the recording run for the duration of the material, in this case something over 40 minutes.  A download would, presumably, complete in a few seconds or minutes.  But it looks like you're stuck here.
